pandas/tests/io/excel/test_xlsxwriter.py

```diff
@@ -5,6 +5,10 @@
 from pandas import DataFrame
 from pandas.util.testing import ensure_clean
 
+from pandas.core.indexes.multi import MultiIndex
```

Comment:
Should revert changes here

Reply:
Ah - forgot about this - will remove. Thanks

Reply:
fixed